languages/csharp/exercises/concept/nullability/.docs/introduction.md

```diff
@@ -0,0 +1,14 @@
+In C# and in many other languages, the `null` literal denotes the abscence of a value.
+
+A *nullable* type is a type that allow `null` values.
+
+There are two main families of types in C#: *reference* and *value*
+types. A variable of a *value* type, contains a direct value in
+memory, while a *reference* typed variable contains a *pointer* to a
+value.
+
+A variable has a *nullable* type if it can contain no value. *Reference* types are nullable by default. 
```

Comment:
> *Reference* types are nullable by default.

I think we should be explaining how C# 8 is changing this.

Reply:
What if we do it the other way around: as the feature is enabled on the csproj level, we could leave this detail and the difference between reference/value types to the `after.md` file.

Reply:
Just pushed a new version with a proposal for that... maybe the concept should be introduced somewhere else, for instance, on an exercise about value/reference parameter passing?

Reply:
>  we could leave this detail and the difference between reference/value types to the after.md file.

I like this. I would like to include it here, but the `after.md` file is perfect for that.